packages/mitmweb-ui/default.nix

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenvNoCC,
+  pkgs,
+  lib,
+  ...
+}: let
+  waypipePort = 1100; # TODO: remove hardcoded port number
+  nmLauncher =
+    pkgs.writeShellScript
+    "mitmweb-ui"
+    ''
+      # Create ssh-tunnel between chromium-vm and ids-vm
+      ${pkgs.openssh}/bin/ssh -i /run/waypipe-ssh/id_ed25519 \
+          -o StrictHostKeyChecking=no \
+          -t ghaf@chromium-vm.ghaf \
+              ${pkgs.openssh}/bin/ssh -M -S /tmp/control_socket \
+              -f -N -L 8081:localhost:8081 ghaf@192.168.100.3
```

Comment:
Please define named variable for `ghaf@192.168.100.3` under `let` and use that here.

Reply:
Resolved by [0e2656b](https://github.com/tiiuae/ghaf/commit/0e2656bf1a00ef6752bcbf2781aa5ae7f204b685).
Also changed the static IP address.